pkgs/esp-idf/default.nix

```diff
@@ -1,4 +1,4 @@
-{ rev ? "v5.1.3"
+{ rev ? "v5.2.2"
 , sha256 ? "sha256-0QsIFOcSx1N15t5po3TyOaNvpzBUfKaFdsRODOBoXCI="
```

Comment:
you likely also want to change this hash.

Reply:
Definitely! I was using this from a local branch that had version overrides, so didn't notice